taxonomic_dist_by_soil_layer/R/taxonomic_dist_soil_layer_R.ipynb

```diff
@@ -0,0 +1,2622 @@
+{
```

Comment:
<p>Has this added any information? <code>biosample_df2</code> and <code>biosample_df3</code> look the same </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microbiomedata/nmdc_notebooks/pull/42/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Good eye.  I think we can skip that API call and simply rename the header from <code>pooling_id_has_input</code> to <code>processed_sample_id</code>   </p>